pkg/ec2pricing/odpricing.go

```diff
@@ -214,6 +215,12 @@ func (c *OnDemandPricing) getRegionForPricingAPI() string {
 			regionDescription = region.Description()
 		}
 	}
+
+	// endpoints package returns European regions with the word "Europe," but the pricing API expects the word "EU."
+	// This formatting mismatch is only present with European regions.
+	// So replace "Europe" with "EU" if it exists in the regionDescription string.
+	regionDescription = strings.Replace(regionDescription, "Europe", "EU", 2)
```

Comment:
Why do we specify 2?

Reply:
I have changed `Replace()` with `ReplaceAll()`